src/main/java/com/github/fge/jsonschema/main/JsonSchemaFactory.java

```diff
@@ -60,7 +60,7 @@
  *     <li>a {@link SyntaxValidator}, to validate schemas;</li>
  *     <li>a {@link JsonValidator}, to validate an instance against a schema;
  *     </li>
- *     <li>a {@link JsonSchema}, to validate instances against a fixed schema.
+ *     <li>a {@link JsonSchemaImpl}, to validate instances against a fixed schema.
```

Comment:
See https://github.com/fge/json-schema-validator/issues/159 it made sense and it was a simple 15 minute patch to make.